joomla/joomla-cms
# [4.0] Global Config: correcting Text filters display

### Summary of Changes 
Following up on https://github.com/joomla/joomla-cms/pull/27684#issuecomment-580189119
`custom-select` should be auto instead of 100%

### Testing Instructions
Display Global Configuration Text Filters tab. Reduce window width.
See before patch. Patch and use npm

### Before patch
![beforeconfig](https://user-images.githubusercontent.com/869724/73441685-a783ca00-4353-11ea-9811-8de891cc2dcb.gif)


### After patch
![afterconfig](https://user-images.githubusercontent.com/869724/73441392-37754400-4353-11ea-8315-27091bf611e2.gif)
